docs/docs/api/webhook.md

```diff
@@ -94,23 +94,52 @@ Subscribes the webhook for the first time or update its parameters.
 }
 ```
 
+## Update
+
+`POST /webhooks.update`
+
+Update the webhook parameters.
+
+**Sample response**
```

Comment:
If this is the response payload then we still lack a sample request payload

Reply:
Sure I will add sample request too.